src/assets/i18n/en.json

```diff
@@ -159,5 +159,31 @@
+    "contactTitle": "Contact Information",
+    "contactText": "If you have any questions or other matters pertaining to the App’s data collection and processing, please direct them to: <br><br>Westfälische Wilhelms-Universität, Institut für Geoinformatik<br>Nina Meyer-Pachur<br>Beauftragte für den Datenschutz<br>Schlossplatz 2<br>48149 Münster<br>+ 49 251 83-22446<br><a href='mailto:datenschutz@uni-muenster.de'>datenschutz@uni-muenster.de</a>",
```

Comment:
The `contactText` could be translated a bit more, see [also here](https://www.uni-muenster.de/de/en/datenschutzerklaerung.html).

- `Westfälische Wilhelms-Universität, Institut für Geoinformatik` → `Westfälische Wilhelms-Universität Münster (WWU) / University of Münster (WWU)`
- `Beauftragte für den Datenschutz` → `data protection officer`